src/microbots/MicroBot.py

```diff
@@ -167,7 +218,7 @@ def _create_environment(self, folder_to_mount: Optional[Mount]):
             )
 
     def _create_llm(self):
-        if self.model_provider == ModelProvider.OPENAI:
+        if self.model_provider in [ModelProvider.OPENAI, ModelProvider.OPENAI_STANDARD]:
```

Comment:
`ModelProvider` is an internal class. To use `ModelProvider.OPENAI_STANDARD`, You need to introduce that enum in the constants.py file.